src/data/thunks.js

```diff
@@ -75,10 +75,8 @@ export function getProctoringSettings() {
       logError('Failed to get exam settings. No exam id.');
       return;
     }
-    dispatch(setIsLoading({ isLoading: true }));
```

Comment:
I think it does because this is one of the possible issues why we had multiple API calls for review policy when we only need to do one. When we put isLoading=true we show spinner and then some of the components get rerendered because of this and because of that we call API more than once. We don't need to set loading for settings I think, we can safely start showing the page while they are being loaded.